src/http/reply.cc

```diff
@@ -49,144 +50,68 @@ namespace http {
 
 namespace status_strings {
 
-const sstring continue_ = "100 Continue";
```

Comment:
This may break existing 3rd party project compilation

Reply:
How come? These constants are local to this compilation unit

Reply:
Ah, it's .cc file, indeed. Disregard this